index.html

```diff
@@ -54,7 +54,11 @@
 
     <div id="EaselView" class="row">
             <div id="ToolbarView" class="col-md-1">
-                tools
+				<button type="button" class="btn btn-default toolbar-btn" id="select-btn"><img height="60px" src="assets/resources/button-images/grab.png"></button>
+				<button type="button" class="btn btn-default toolbar-btn" id="draw-btn"><img height="60px" src="assets/resources/button-images/pencil.png"></button>
+				<button type="button" class="btn btn-default toolbar-btn" id="erase-btn"><img height="60px" src="assets/resources/button-images/eraser.png"></button>
+				<button type="button" class="btn btn-default toolbar-btn" id="measure-btn"><img height="60px" src="assets/resources/button-images/ruler.png"></button>
```

Comment:
like, apparently it's supposed to measure edges? I really don't think we need to worry about it now
________________________________
From: mmou [notifications@github.com]
Sent: Wednesday, April 15, 2015 12:58 AM
To: mmou/tacit
Cc: Megan R O'Leary
Subject: Re: [tacit] Toolbar (#7)


In index.html<https://github.com/mmou/tacit/pull/7#discussion_r28393406>:

> @@ -54,7 +54,11 @@
>
>      <div id="EaselView" class="row">
>              <div id="ToolbarView" class="col-md-1">
> -                tools
> +                             <button type="button" class="btn btn-default toolbar-btn" id="select-btn"><img height="60px" src="assets/resources/button-images/grab.png"></button>
> +                             <button type="button" class="btn btn-default toolbar-btn" id="draw-btn"><img height="60px" src="assets/resources/button-images/pencil.png"></button>
> +                             <button type="button" class="btn btn-default toolbar-btn" id="erase-btn"><img height="60px" src="assets/resources/button-images/eraser.png"></button>
> +                             <button type="button" class="btn btn-default toolbar-btn" id="measure-btn"><img height="60px" src="assets/resources/button-images/ruler.png"></button>


can someone explain to me what 'ruler tool' does exactly?

—
Reply to this email directly or view it on GitHub<https://github.com/mmou/tacit/pull/7/files#r28393406>.